### PR TITLE
Layer-2 vulnerabilities in various specifications

### PR DESCRIPTION
**Pull request by sei-vsarvepalli**
These vulnerabilities were reported by Etienne Champetier @champtar  https://github.com/champtar <champetier.etienne@gmail.com> via CERT/CC to multiple vendors.

### CODE REVIEW

File: 2021/27xxx/CVE-2021-27853.json [hunk 1]
```diff
@@ -4,15 +4,118 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27853.json:4]: Invalid source.

File: 2021/27xxx/CVE-2021-27854.json [hunk 2]
```diff
@@ -4,15 +4,126 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27854.json:4]: Invalid source.

File: 2021/27xxx/CVE-2021-27861.json [hunk 3]
```diff
@@ -4,15 +4,134 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27861.json:4]: Invalid source.

File: 2021/27xxx/CVE-2021-27862.json [hunk 4]
```diff
@@ -4,15 +4,109 @@
     "data_version": "4.0",
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27862.json:4]: Invalid source.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/27xxx/CVE-2021-27853.json [hunk 5]
```diff
@@ -4,15 +4,123 @@
...
```
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27853.json:4]: Invalid source.

File: 2021/27xxx/CVE-2021-27854.json (hunk 2 above)
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27854.json:4]: Invalid source.

File: 2021/27xxx/CVE-2021-27861.json (hunk 3 above)
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27861.json:4]: Invalid source.

File: 2021/27xxx/CVE-2021-27862.json (hunk 4 above)
Comment (cve-team) [inline 2021/27xxx/CVE-2021-27862.json:4]: Invalid source.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by sei-vsarvepalli** [comment]
These hopefully are fixed now.

Thanks

File: 2021/27xxx/CVE-2021-27853.json [hunk 6]
```diff
@@ -4,15 +4,123 @@
...
     "CVE_data_meta": {
         "ID": "CVE-2021-27853",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
+        "ASSIGNER": "cert@cert.org",
+        "DATE_PUBLIC": "20220927T00:00:00.000000Z",
```
Comment (champtar) [inline 2021/27xxx/CVE-2021-27853.json:8]: You are a bit early ;)

**Comment by sei-vsarvepalli** [comment]
Closing this commit and pushing dates out to September 27th as requested for public release.

**Pull request closed without merging**